tools/finish_release.py

```diff
@@ -216,23 +167,19 @@ def promote_snaps(version):
                 print(e.stdout)
                 raise
 
-
 def main(args):
     parsed_args = parse_args(args)
 
-    github_access_token_file = parsed_args.githubpat
-    github_access_token = open(github_access_token_file, 'r').read().rstrip()
-
-    with tempfile.TemporaryDirectory() as tempdir:
-        version = download_azure_artifacts(tempdir)
-        # Once the GitHub release has been published, trying to publish it
-        # again fails. Publishing the snaps can be done multiple times though
-        # so we do that first to make it easier to run the script again later
-        # if something goes wrong.
-        if parsed_args.publish_snaps:
-            promote_snaps(version)
-        if parsed_args.publish_windows:
-            create_github_release(github_access_token, tempdir, version)
+    css = parsed_args.css
+
+    # Once the GitHub release has been published, trying to publish it
+    # again fails. Publishing the snaps can be done multiple times though
+    # so we do that first to make it easier to run the script again later
+    # if something goes wrong.        
+    if parsed_args.publish_snaps:
+        promote_snaps(version)
```

Comment:
Have you not had a chance to test this yourself yet? This script crashes for me on this line because `version` is no longer defined. To fix that, maybe we could make a simple function called something like `fetch_version_number` that contains the code to grab the version number from Azure Pipelines?